include/modem/location.h

```diff
@@ -109,10 +124,26 @@ struct location_datetime {
 	uint16_t ms;
 };
 
+#if defined(CONFIG_LOCATION_DATA_DETAILS)
+/** Location details for GNSS. */
+struct location_data_details_gnss {
+	/** Whether these details are valid. */
```

Comment:
```suggestion
	/** Checking whether these details are valid. */
```

Reply:
This sounds a bit odd. I'd say `Indicates where these details are valid.` if it's not OK as it is.

Reply:
I was supposed to say `Indicates whether these details are valid.`
Is that fine?